### ISSUE TITLE
How to generate mix precision engines?

### ISSUE THREAD

**Issue by Justin020718** [NVIDIA/TensorRT#4027]
## Description

  I'm trying to accelerate inference with TensorRT, the fp16 model works fine, but when I'm trying to generate int8 model, the accuracy was not good. I'm wondering if I can generate some mix-precision models where some layers are in fp16 and most layers are in int8? I'll appreciate it if someone could offer some guidance. Thank you for your time and patience.

## Environment

**TensorRT Version**:
10.2.0.19
**NVIDIA GPU**:
RTX-4090
**NVIDIA Driver Version**:
newest
**CUDA Version**:
12.5
**CUDNN Version**:
newest

Operating System:
windows 11
Python Version (if applicable):
3.8
Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by lix19937** [comment]
`trtexec --layerPrecisions=spec  --layerOutputTypes=spec --fp16 --int8 `

**Comment by Justin020718** [comment]
> `trtexec --layerPrecisions=spec --layerOutputTypes=spec --fp16 --int8 --calib=<file>`
@lix19937 
Thank you for your rapid reply! However, how can I figure out which layers may need higher precison?

**Comment by lix19937** [comment]
Find the sensitive layer through iteration.

**Comment by lix19937** [comment]
YES @Egorundel
